controllers/util/poolutil.go

```diff
@@ -30,6 +30,10 @@ const (
 
 	CsiIBMBlockDriver = "block.csi.ibm.com"
 	CsiIBMBlockScPool = "pool"
+
+	SecretName              = "csi.storage.k8s.io/provisioner-secret-name"
+	SecretNamespace         = "csi.storage.k8s.io/provisioner-secret-namespace"
```

Comment:
maybe `SecretNameKey` /`SecretNamespaceKey` would be more accurate?

Reply:
fixed